app/travel_analytics/ui5.yaml

```diff
@@ -30,3 +34,11 @@ server:
         baseUri: http://localhost:4004/analytics
         username: admin # dummy credentials for local testing
         password: admin # dummy credentials for local testing
+    - name: fiori-tools-appreload
```

Comment:
`ui5-middleware-simpleproxy` probably not needed?

Reply:
@c-kobo I created a PR into this branch https://github.com/SAP-samples/cap-sflight/pull/1298 where the apps load from the watch scripts at the root.